homeassistant/components/flipr/translations/fr.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Please do not add additional translations in the initial PR. Translations can be managed via lokalise after this PR has been merged.

Reply:
OK, correction done via a new commit and push.